.github/workflows/dev_cd.yaml

```
@@ -1,49 +1,60 @@
 name: Dev CD
```

Also rename the file to cd.yaml.

```suggestion
name: CD
```